### PR TITLE
Prettier pre commit

### PR DESCRIPTION
**Pull request by eskilgh**
Switching to pretty-quick for pre-commits as prettier --check seems to not work well for windows users with crlf

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```
Comment (brikendr) [inline .husky/pre-commit:1]: Husky?

File: package.json [hunk 2]
```diff
@@ -47,20 +47,15 @@
     "prepublish-package": "node ./scripts/rmLibDir",
     "publish-package": "npm run prepublish-package && npm run build && npm publish",
     "compodoc": "compodoc -w -p tsconfig.json -s -n \"huddly-sdk\" -d \"docs\" --minimal --customFavicon \"docs_style/assets/imgs/favicon.ico\" --disableGraph --disableInternal --hideGenerator --includesName \"Additional Information\"  --includes examples -a docs_style/assets -y docs_style",
-    "prepare": "npm run build",
+    "prepare": "npm run build && husky install",
```
Comment (brikendr) [inline package.json:50]: This will not work if `husky` is a dev dependency.

Comment (brikendr) [reply package.json:50]: What does `husky install` do?

**Review by brikendr — CHANGES_REQUESTED** [review]
Not sure what the benefits of using `husky` are but you need to make some changes to make this work.

File: .husky/pre-commit (hunk 1 above)
Comment (eskilgh) [reply .husky/pre-commit:1]: It's a git hook package. Afaik it's the best way for running pretty-quick in pre-commit.
Also it has zero dependencies, so it shouldn't be too bad to add as a dev dependency.

File: package.json (hunk 2 above)
Comment (eskilgh) [reply package.json:50]: I'll look into it, I think I can just disable this for CI. Is there an env variable or something that makes it easy to tell if I'm currently in a CI environment?

**Pull request closed without merging**